recipes/gtk4/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+pango:
+  - 1.48
```

Comment:
ah great! where is that PR?

Reply:
`conda-forge-pinning` has already been updated with this change, so can I drop the pin now already?

Reply:
It's just a migrator for now, and those don't get applied on staged-recipes. I don't know if you can manually apply the migrator instead of using `conda_build_config.yaml` (I don't think so?). Still fine to have this I think, but we can remove it immediately once the feedstock is created.